spec/integration/inventory_spec.rb

```diff
@@ -25,24 +25,46 @@ module Hawkular::Inventory::RSpec
     end
   end
 
-  describe 'Inventory', vcr: { decode_compressed_response: true } do
+  describe 'Inventory' do
     before(:all) do
       @creds = {
         username: 'jdoe',
         password: 'password'
       }
       @client = Hawkular::Inventory::InventoryClient.create(credentials: @creds)
-      @state = {
-        hostname: 'localhost.localdomain',
-        feed: nil
-      }
+      options = { decode_compressed_response: true }
+      options[:record] = :all if ENV['VCR_UPDATE'] == '1'
+      VCR.use_cassette('Inventory/Helpers/get_feeds', options) do
+        feeds = @client.list_feeds
+        @state = {
+          hostname: 'dhcp-10-40-1-43.brq.redhat.com',
```

Comment:
> Do I get this right, that the client still talks to Hawkular on e.g. localhost

nope, it talks to hawkular only if `VCR_OFF=1` (use case: trying if it still works) or `VCR_UPDATE=1` (use case: updating the cassettes/templates, perhaps the hawk api was changed)

> ..and that this 'state' is a placeholder?

Yeah, it's kind of a placeholder so that the `feed_id` don't have to be hardcoded to a value that is stored in the cassette. Actually the `Inventory/Helpers/get_feeds` cassette is the only cassette with the `feed_id` all the others have it parametrized by the ERB.

wrt `hostname='dhcp-10-40-1-43.brq.redhat.com'`, it's probably wrong how it is right now, perhaps I should have removed this check, because it's different on each environment and this makes updating the cassetes or running the tests online a pain.